sass/layouts/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  position: fixed;
+  z-index: 100;
+  width: 100%;
+  height: auto;
+  color: $color-logo;
+  background: $color-bg;
+  border-bottom: 6px solid $color-bg-border;
+  @include media-768 {
+    border-bottom: none;
+  }
+}
+
+.logo-center {
+  position: absolute;
+  @include media-768 {
+    position: relative;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+}
+
+.header__main {
+  @include flex();
+  width: 100%;
+}
+
+.header__navigation {
+  margin: 42px 12px 33px;
+  @include media-768 {
+    display: none;
+  }
+  @include media-375 {
+    display: none;
+  }
+  @include media-320 {
+    display: none;
```

Comment:
If you remove @include media-375 and @include media-320 blocks here, will something change?

Reply:
DONE!